js/packages/web/src/components/AuctionCard/index.tsx

```diff
@@ -975,6 +968,8 @@ export const AuctionCard = ({
             </Button>
           )}
 
+          {crossmintBtn}
```

Comment:
don't we still need to check for a crossmintClientId and whether the auction is a Buy Now?

Reply:
Fixed!